sklearn-example/README.md

```diff
@@ -0,0 +1,67 @@
+# Log Files
+
+This is an example of an application that classifies streams of hand-written digits. This example is discussed in the [A scikit-learn pipeline in Wallaroo](http://blog.wallarolabs.com/) blog post. For details on how the application works, please refer to the blog post.
```

Comment:
blog post link should be:

https:/blog.wallaroolabs.com/2018/02/a-scikit-learn-pipeline-in-wallaroo/